### PR TITLE
Water tank fixes and staples crafting

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: You can remove the top from a water tank and put chemicals in it
fix: You can craft staples
add: New tank sprites
/:cl:

https://vgy.me/odfQJI.png

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Don't forget to put your name after the first CL

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -18,6 +18,26 @@
 			boom()
 
 /obj/structure/reagent_dispensers/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W, /obj/item/weapon/screwdriver))
+		if(src.icon_state == "water" && !src.is_open_container())
+			src.icon_state = "wateropen"
+			user << "<span class='notice'>You unscrew and remove the lid from [src].</span>"
+			src.container_type = OPENCONTAINER
```
Comment (JohnGinnane) [inline code/modules/reagents/reagent_dispenser.dm]: Might be better to do `container_type |= OPENCONTAINER`

File: code/modules/reagents/reagent_dispenser.dm [hunk 2]
```diff
@@ -18,6 +18,26 @@
...
+			user << "<span class='notice'>You unscrew and remove the lid from [src].</span>"
+			src.container_type = OPENCONTAINER
+			src.update_icon()
+			playsound(src, 'sound/items/Deconstruct.ogg', 20, 1)
+			desc = "A water tank. It's top has been removed."
+			return
+		if(src.icon_state == "wateropen" && src.is_open_container())
+			src.icon_state = "water"
+			user << "<span class='notice'>You reattach and screw the [src]'s lid into place.</span>"
+			src.container_type = null
```
Comment (JohnGinnane) [inline code/modules/reagents/reagent_dispenser.dm]: And then `container_type &= ~OPENCONTAINER` here

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

**Comment by KittyKathy** [comment]
So does this only work for water tanks? what about welding tanks, high capacity water tanks that botany gets and chemical tanks from atmos does tg even have chemical tanks though? I know pre rebase had it but that was rarely used.

**Comment by tortellinitony** [comment]
This only works for water tanks, and tg does not even have chemical tanks.

**Comment by KittyKathy** [comment]
Why not make it work for all of them besides the problem of only the water tank having a sprite. I don't think they had a sprite pre rebase though did they?

**Review by j-mie — APPROVED** [review]

**Pull request merged**